OutOfSchool/OutOfSchool.WebApi/Hubs/ChatWorkshopHub.cs

```diff
@@ -167,7 +168,17 @@ public async Task SendMessageToOthersInGroupAsync(string chatNewMessage)
             // Add Provider's connections to the Group.
             var workshops = await workshopRepository.GetByFilter(w => w.Id == chatMessageWorkshopCreateDto.WorkshopId, "Provider").ConfigureAwait(false);
             var workshop = workshops.SingleOrDefault();
-            await AddConnectionsToGroupAsync(workshop.Provider.UserId, createdMessageDto.ChatRoomId.ToString()).ConfigureAwait(false);
+
+            if (workshop != null)
+            {
+                await AddConnectionsToGroupAsync(workshop.Provider.UserId, createdMessageDto.ChatRoomId.ToString()).ConfigureAwait(false);
+            }
+            else
+            {
+                logger.LogWarning("Workshop is null. Unable to add connections to group.");
+
+                return;
+            }
```

Comment:
(Code style) Please use "fail fast" approach - invert this `if` condition and move await `AddConnections...` under the `if`. Thus you'll have `workshop != null` related logic not split into 2 pieces (as now it is both inside `if` clause and rest of the method).